packages/node/tsconfig.json

```diff
@@ -4,7 +4,8 @@
     "sourceMap": true,
     "tsBuildInfoFile": "dist/.tsbuildinfo",
     "rootDir": "src",
-    "outDir": "./dist"
+    "outDir": "./dist",
+    "strict": true
```

Comment:
The strict option should be able to be moved from all packages to the root config.